docs/mast/mast.rst

```diff
@@ -575,6 +456,110 @@ Downloading data products from S3:
    ...
    >>> Observations.disable_cloud_dataset()
 
+   
+
+Mission Searches
+================
+
+Mission-Specific Search Queries
+-------------------------------
+
+These queries allow for searches based on mission-specific metadata for a given
+data collection.  Currently it provides access to a broad set of Hubble Space
+Telescope (HST) metadata, including header keywords, proposal information, and
+observational parameters.  The available metadata includes all information that
+was previously available in the original HST web search form, and are present in
+the current `Mission Search interface <https://mast.stsci.edu/search/ui/#/hst>`__.
+
+**Note:**, this API interface does not yet support data product download, only
+metadata earch access.
+
+An object of MastMissions class is instantiated with a default mission of 'hst' and 
+default service set to 'search'.
+
+.. doctest-remote-data::
+
+   >>> from astroquery.mast.missions import MastMissions
+   >>> missions = MastMissions()
+   >>> missions.mission
+   'hst'
+   >>> missions.service
+   'search'
+
+The missions object can be used to search metadata using by sky position, or other criteria.
+The keyword arguments can be used to specify output characteristics like selec_cols and
+sort_by and conditions that filter on values like proposal id, pi last name etc.
+The available column names for a mission are returned by the
+`~astroquery.mast.MastMissionsClass.get_column_list` function.
+
+.. doctest-remote-data::
+   
+   >>> from astroquery.mast.missions import MastMissions
+   >>> missions = MastMissions(mission='hst')
+   >>> columns = missions.get_column_list()
+
+For positional searches, the columns "ang_sep", "sci_data_set_name", "search_key" and "search_position"
+will always be included, in addition to any columns specified using "select_cols". For non-positional
+searches, "search_key" and "sci_data_set_name" will always be included, in addition to any columns
+specified using "select_cols". 
+
+.. doctest-remote-data::
+   
+   >>> from astroquery.mast.missions import MastMissions
+   >>> from astropy.coordinates import SkyCoord
+   ...
+   >>> missions = MastMissions(mission='hst')
+   >>> regionCoords = SkyCoord(210.80227, 54.34895, unit=('deg', 'deg'))
+   >>> results = missions.query_region(regionCoords, 3, sci_pep_id=12556,
```

Comment:
the changes from the other PR are not got propagated here, e.g. this would fail without adding `radius`, etc.

Reply:
Gah! That'll teach me to do a "quick update" late at night. Sorry. I will fix this when I get a space moment.